.gitignore

```diff
@@ -1,4 +1,4 @@
-﻿# Project
+# Project
```

Comment:
I can't tell what's changed in this line.

Reply:
It's actually some BOM(Byte Order Mark) with UTF-8 encoding. I tried multiple times to make it pass the pre-commit check.

Reply:
I could reverse this change and the line below if necessary.

Reply:
> I can't tell what's changed in this line.

After checking with `pre-commit run --files .gitignore` in main branch, we could see that the current `.gitignore` file has a byte-order marker which couldn't pass the pre-commit. It didn't raise in other PRs because none of them updated the file.

So I think it's OK to update it in this PR.